packages/onboarding/src/flow-progress/use-flow-progress.ts

```diff
@@ -20,11 +20,12 @@ const flows: Record< string, { [ step: string ]: number } > = {
 	newsletter: {
 		intro: 0,
 		user: 0,
-		newsletterSetup: 1,
-		domains: 2,
-		'plans-newsletter': 3,
-		subscribers: 4,
-		launchpad: 5,
+		newsletterType: 1,
+		newsletterSetup: 2,
+		domains: 3,
+		'plans-newsletter': 4,
+		subscribers: 5,
+		launchpad: 6,
```

Comment:
This makes "progress" bar work in the flow. It's cumbersome how this flow-specific definition is in some package, instead of keeping this at the same place where newsletter flow is defined.

Reply:
Agreed about it being non-ideal the way it is separated. But I'd think if we were going to address that, probably different PR. And given other priorities, my guess is this won't rise high enough on the list for us to refactor it.